src/arkode/arkode_mristep.c

```diff
@@ -2258,9 +3615,13 @@ int mriStep_ComputeInnerForcing(SUNDIALS_MAYBE_UNUSED ARKodeMem ark_mem,
   cvals = step_mem->cvals;
   Xvecs = step_mem->Xvecs;
 
+  /* Set inner forcing time normalization constants */
+  step_mem->stepper->tshift = t0;
+  step_mem->stepper->tscale = tf - t0;
+
   /* compute inner forcing vectors (assumes cdiff != 0) */
   nstore = 0;
-  for (j = 0; j < stage; j++)
+  for (j = 0; j < SUNMIN(stage, step_mem->stages); j++)
```

Comment:
Yes, because the input `stage` value can now equal `step_mem->stages`, and the tables `W` and `G` have `step_mem->stages+1` rows each, to include the embedding.